lib/main_common.pm

```diff
@@ -1098,8 +1098,9 @@ sub load_consoletests {
 
     loadtest "locale/keymap_or_locale";
     if (is_sle && !get_var('MEDIA_UPGRADE') && !get_var('ZDUP') && is_upgrade && !is_desktop && !get_var('INSTALLONLY')) {
+	loadtest "console/supportutils";
         loadtest "console/check_upgraded_service";
-        loadtest "console/supportutils";
+	#loadtest "console/supportutils";
```

Comment:
I guess you forgot this

Reply:
Yes, it is for debug. I just updated code. Thanks.